sentinelsat/sentinel.py

```diff
@@ -1384,6 +1384,8 @@ def default_converter(x):
                             name = "link_" + p["rel"]
                         product_dict[name] = p["href"]
                 else:
+                    if "content" not in p and "str" not in p:
+                        continue
```

Comment:
Was this really the line where you wanted to add this check? 

What is `p` supposed to be at this point, since it is only defined in the first branch on line 1381 (`for p in properties`)?

Reply:
I see you already fixed it.